common/Log.hpp

```diff
@@ -247,43 +247,37 @@ inline constexpr void logPrefix(std::ostream&) {}
 #define STRINGIFY(X) #X
 #define STRING(X) STRINGIFY(X)
 
-#define LOG_END(LOG, FILEP)                                                                        \
-    do                                                                                             \
-    {                                                                                              \
-        if (FILEP)                                                                                 \
-            LOG << "| " LOG_FILE_NAME(__FILE__) ":" STRING(__LINE__);                              \
-        LOG.flush();                                                                               \
-    } while (false)
-
 #ifdef __ANDROID__
 
-#define LOG_BODY_(LOG, PRIO, LVL, X, FILEP)                                                        \
-    char b_[1024];                                                                                 \
-    std::ostringstream oss_(Log::prefix<sizeof(b_) - 1>(b_, LVL), std::ostringstream::ate);        \
-    logPrefix(oss_);                                                                               \
-    oss_ << std::boolalpha << X;                                                                   \
-    LOG_END(oss_, FILEP);                                                                          \
-    ((void)__android_log_print(ANDROID_LOG_DEBUG, "coolwsd", "%s %s", LVL, oss_.str().c_str()))
+#define LOG_LOG(LOG, PRIO, LVL, STR)                                                               \
+    ((void)__android_log_print(ANDROID_LOG_DEBUG, "coolwsd", "%s %s", LVL, STR.c_str()))
 
 #else
 
-#define LOG_BODY_(LOG, PRIO, LVL, X, FILEP)                                                        \
+#define LOG_LOG(LOG, PRIO, LVL, STR)                                                               \
+    LOG.log(Poco::Message(LOG.name(), STR, Poco::Message::PRIO_##PRIO))
```

Comment:
This PRIO_##PRIO trick at first caused a problem for me as for some reason the Xcode project defined a macro DEBUG=1. After I fixed that to use ENABLE_DEBUG, no problem.